src/test/java/com/iota/iri/MilestoneTrackerTest.java

```diff
@@ -0,0 +1,171 @@
+package com.iota.iri;
+
+
+import com.iota.iri.conf.ConfigFactory;
+import com.iota.iri.conf.IotaConfig;
+import com.iota.iri.controllers.TipsViewModel;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.hash.SpongeFactory;
+import com.iota.iri.model.TransactionHash;
+import com.iota.iri.network.TransactionRequester;
+import com.iota.iri.storage.Tangle;
+import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
+import com.iota.iri.utils.Converter;
+import com.iota.iri.zmq.MessageQ;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+
+public class MilestoneTrackerTest {
+
+    private static final TemporaryFolder dbFolder = new TemporaryFolder();
+    private static final TemporaryFolder logFolder = new TemporaryFolder();
+    private static Tangle tangle;
+    private static MilestoneTracker milestoneTracker;
+
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        tangle.shutdown();
+        dbFolder.delete();
+    }
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+
+
```

Comment:
While changing this remove all those whitespaces.
I usually don't mind them too much but you went overboard here.